samples/v1.3/Scenarios/FoodOrderv2.json

```diff
@@ -1,151 +0,0 @@
-{
-	"$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
-	"type": "AdaptiveCard",
-	"version": "1.0",
-	"body": [
-		{
-			"type": "TextBlock",
-			"text": "Your registration is almost complete",
```

Comment:
hmm - do you have the rebasing done right ? I dont think we want to delete these samples from main.